README.rst

```diff
@@ -85,20 +85,64 @@ How to use this tool
 1. git clone this repo:
 
    ::
-   
+
       git clone https://github.com/ios-xr/iosxrv-x64-vbox.git
 
-2. Install VirtualBox and Vagrant (see guide below).
+2. Install VirtualBox, Vagrant and socat (see guide below).
 3. Download the appropriate ISO file, e.g. ``iosxrv-fullk9-x64.iso``
 4. Generate the VirtualBox box:
 
    ::
-   
-      ./iosxrv-x64-vbox/iosxr_iso2vbox.py -i iosxrv-fullk9-x64.iso
 
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-How to install Vagrant and VirtualBox
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+      ./iosxrv-x64-vbox/iosxr_iso2vbox.py iosxrv-fullk9-x64.iso
+
+5. Example with verbosity off
+
+   ::
+      iosxrv-x64-vbox/iosxr_iso2vbox.py iosxrv-fullk9-x64.iso
```

Comment:
I think you need a blank line after the `::` - this isn't rendering properly on GitHub (https://github.com/ios-xr/iosxrv-x64-vbox/blob/cfad89e45ef99d2b27b0589cc0e4e289185fefaa/README.rst)